<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserSolverCustomForm</class>
 <widget class="QWidget" name="UserSolverCustomForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>8000</width>
    <height>5000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxUserSolverOptions">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>600</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1500</width>
       <height>1500</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Define model options for the groundwater flow module.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="title">
      <string>User Solver with custom equations</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QLabel" name="labelUserSolverCustom">
           <property name="minimumSize">
            <size>
             <width>541</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>541</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Input user solver routine</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="lineEditSolverFile"/>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonSolverFile">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Select a local mesh directory.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="resources_pages.qrc">
               <normaloff>:/new/prefix1/icons/22x22/folder-open.png</normaloff>:/new/prefix1/icons/22x22/folder-open.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonSolverFileClear">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Clean the selected mesh directory.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="resources_pages.qrc">
               <normaloff>:/new/prefix1/icons/22x22/edit-clear.png</normaloff>:/new/prefix1/icons/22x22/edit-clear.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="pushButtonSolverFileEdit">
           <property name="text">
            <string>Create/Edit user solver routine</string>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>customUserSolver_label</zorder>
        <zorder>customUserSolver_label_2</zorder>
        <zorder>labelMeshInput</zorder>
        <zorder>lineEdit</zorder>
        <zorder>toolButtonMeshDir</zorder>
        <zorder>toolButtonMeshDirClear</zorder>
        <zorder>pushButton</zorder>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="customUserSolver_label_2">
        <property name="text">
         <string>This is done through a user routine. See the example in SRC/examples/cs_user_solver-heat-equation.c</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="customUserSolver_label">
        <property name="text">
         <string>Use the pre/post-processing and parallelisation capabilities of Code_Saturne to compute your own equations.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources_pages.qrc"/>
 </resources>
 <connections/>
</ui>
